templates/default/customer/customertickets.html

```diff
@@ -0,0 +1,121 @@
+<!--// $Id$ //-->
+
+{css}
+{js}
+
+{tab_container id="customertickets" label="Customer tickets"}
+
+	{tab_header content_id="ticketspanel"}
+		{tab_header_cell icon="lms-ui-icon-helpdesk"}
+			<strong>{trans("Customer tickets")}</strong>
+		{/tab_header_cell}
+		{tab_header_cell}
+			{button type="link" tip="New Ticket" icon="add" href="?m=rtticketadd&customerid={$customerinfo.id}"}
+			{button type="link" tip="Show all customer tickets" icon="search" href="?m=rtsearch&id={$customerinfo.id}"}
+		{/tab_header_cell}
+	{/tab_header}
+
+	{tab_contents id="ticketspanel"}
+		{tab_table}
+			<div class="lms-ui-tab-table-row header">
+				<div class="lms-ui-tab-table-wrapper id-type col-4">
```

Comment:
@interduo wiesz o co tu w ogóle chodzi? Podpowiadam: ``col-4` oznacza, że ostatecznie ten kontener może posiadać wewnątrz 4 kolumny w pionie, a wewnątrz jakoś nie widzę czterech tylko dwie. Nie ma tu żadnej filozofii - popatrz innych, działających boksach jak to działa i porównaj z kodem źródłowym. W pliku less/css masz opis co kiedy ma się zapadać z wiesza do kolumny.

Reply:
Dziękuję za wytłumaczenie. Zrozumiałem szybko jak to ogarnąć. Puściłem commit do tego.